tutorials/3d/environment_and_post_processing.rst

```diff
@@ -337,12 +337,20 @@ interesting glow patterns:
 .. image:: img/environment_glow_layers2.png
 
 Finally, as the highest layers are created by stretching small blurred images,
-it is possible that some blockiness may be visible. Enabling **Bicubic Upscaling**
+it is possible that some blockiness may be visible. Enabling **Bicubic Upscale**
 gets rids of it, at a minimal performance cost.
-*Note that this is effective only when using the GLES3 backend.*
 
 .. image:: img/environment_glow_bicubic.png
 
+.. note::
+
+    When using the GLES2 backend, bicubic upscaling is only supported on
+    graphics cards that provide the ``GL_EXT_gpu_shader4`` extension. Nearly all
+    desktop graphics cards provide this extension, but not all mobile GPUs do.
+
+    Also, since GLES2 does not use high dynamic range rendering, you may need to use
+    different glow intensity values to get a good result.
```

Comment:
This could be improved to be more precise, but I don't know what should be changed exactly.